master.iml

```diff
@@ -4,6 +4,15 @@
     <facet type="Spring" name="Spring">
```

Comment:
Looks like IDE config file and should be added to .gitignore

Reply:
Added to .gitignore

Reply:
It should be removed from repository, isn't it?

Reply:
Has been deleted :)